superset-frontend/src/dashboard/components/gridComponents/ChartHolder.jsx

```diff
@@ -183,6 +200,29 @@ class ChartHolder extends React.Component {
         ? parentComponent.meta.width || GRID_MIN_COLUMN_COUNT
         : component.meta.width || GRID_MIN_COLUMN_COUNT;
 
+    let fullStyle = {};
+    let w = 0;
+    let h = 0;
+
+    if (this.state.isFullSize) {
+      fullStyle = {
```

Comment:
instead of applying style to this inline, couldn't we add a new css class to the component then define the style in a less file?

Reply:
I tried to do that
![image](https://user-images.githubusercontent.com/6887197/75914705-3b4d2800-5e23-11ea-8824-dc903d842e61.png)
![image](https://user-images.githubusercontent.com/6887197/75914729-456f2680-5e23-11ea-85cf-59a26afdc1c3.png)
But it shows the following warnings:
![image](https://user-images.githubusercontent.com/6887197/75914681-2f616600-5e23-11ea-9c77-be3b40576c86.png)

I tried different things but nothing work.

Reply:
css doesn't use quotes, try removing the quotes (and the `!important`s)

Reply:
Done, it worked... c6f99a896a1995d6cd7e214b3d6df4d20bfee288